app/admin/adoptions.rb

```diff
@@ -0,0 +1,122 @@
+include ActiveAdminHelpers
```

Comment:
I am wondering why is this file showing up 🤔 since I thought this was merged into `master` and we merged `master` into both these branches 🤷‍♂️

Reply:
:dizzy_face: